lib/writer/object_model_writer.dart

```diff
@@ -147,6 +179,9 @@ class ObjectModelWriter {
         if (c++ > 0) sb.write(' ^\n');
         sb.write('      ${field.name}.hashCode');
       });
+      if (jsonModel.extend != null) {
+        sb.write(' ^ \n      super.hashCode');
```

Comment:
This assumes that the model being extended implements hashCode

Reply:
hashCode has a default implementation, what do you suggest we do here?

Reply:
It has, but it does not make sense to use it if it is the default one, since that one is based on identity instead of value. I don't see any good solution for this though. I was also wondering how badly we can break this by passing a custom model as the parent class, since we don't know the constructor for those...
@vanlooverenkoen ignore?

Reply:
Ignore it is

Reply:
Hmmm, unless you have a tagging interface. But then it is still your responsibility to return true and 1. Good nough